cmd/ingress-operator/render.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
+)
+
+func render(dir string, prefix string) error {
+	files := []string{
+		manifests.CustomResourceDefinitionManifest,
+		manifests.NamespaceManifest,
+	}
+
+	if dir == "-" {
+		for _, file := range files {
+			fmt.Printf("# file: %s\n", filepath.Base(file))
+			fmt.Println(manifests.MustAssetString(file))
+			fmt.Println("---")
+		}
+		return nil
+	}
+
+	if err := os.MkdirAll(dir, 0666); err != nil {
```

Comment:
Wouldn't `0755` make more sense?

Reply:
Yes, this actually explains a perm issue I accidentally worked around. Thanks!

Reply:
Also fixed the perms of the yaml itself

Reply:
> Also fixed the perms of the yaml itself

What was wrong with those?  We don't want the executable bit (odd bit) on the yaml file, do we?

Reply:
Whoops. Should be 0644, right?

Reply:
Re-fixed